usedHttp.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
+
+void makeRequest() async{
+    var response = await http.get('https://example.com/');
+    // If the http request is successful the statusCode will be 200
+    if (response.statusCode == 200){
+      var htmlToParse = response.body;
+      print(htmlToParse);
+    }
+}
+
+dynamic main(){
+  makeRequest();
```

Comment:
Could you modify `makeRequest` into `downloadBody` such that you can use it like so?

```dart
void main() {
  print(downloadBody("http://www.example.com"));
}
```

This will make `main` more readable.